dotnet/samples/KernelSyntaxExamples/Example76_TextEmbedding.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Microsoft.SemanticKernel.Text;
+using RepoUtils;
+using SharpToken;
+
+public static class Example76_TextEmbedding
+{
+    public static async Task RunAsync()
```

Comment:
Please use xUnit IntegrationTests approach.

Reply:
Updated, let me know how that looks.